fastlane.gemspec

```diff
@@ -99,7 +99,7 @@ Gem::Specification.new do |spec|
   spec.add_dependency('optparse', '>= 0.1.1', '< 1.0.0') # Used to parse options with Commander
   spec.add_dependency('plist', '>= 3.1.0', '< 4.0.0') # Needed for set_build_number_repository and get_info_plist_value actions
   spec.add_dependency('rubyzip', '>= 2.0.0', '< 3.0.0') # fix swift/ipa in gym
-  spec.add_dependency('security', '= 0.1.3') # macOS Keychain manager, a dead project, no updates expected
+  spec.add_dependency('security', '= 0.1.5') # macOS Keychain manager, a dead project, no updates expected
```

Comment:
well so much for the "no updates expected".

I am also waiting for https://github.com/fastlane-community/security/pull/12 to be merged and released :)